addons/website_blog/static/src/js/blog_tag_add.js

```diff
@@ -0,0 +1,336 @@
+odoo.define('website_blog.blog_post_tag.add', function (require) {
+'use strict';
+
+const {qweb, _t} = require('web.core');
+const options = require('web_editor.snippets.options');
+const {Class: EditorMenuBar} = require('web_editor.editor');
+
+options.registry.TagBlogPostSelect = options.Class.extend({
+    xmlDependencies: (options.Class.prototype.xmlDependencies || []).concat(['/website_blog/static/src/xml/website_blog_tag.xml']),
+    /**
+     * @override
+     */
+    willStart: async function () {
+        const _super = this._super.bind(this);
+        this.blogPostId = this.$target.data('blog-id');
+        this.addList = [];
+        this.unlinkList = [];
+        this.isEditing = false;
+        this.trigger_up('set_blogPost', {
+            blogPostId: this.blogPostId,
+        });
+        await this._getTags();
+        return _super(...arguments);
+    },
+
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * @see this.selectClass for params
+     */
+    editList: function (previewMode, widgetValue, params) {
+        this.isEditing = true;
+        this._toggleEditUI();
+        this.rerender = true;
+    },
+    /**
+     * @see this.selectClass for params
+     */
+    saveTagList: function (previewMode, widgetValue, params) {
+        this.isEditing = false;
+        this._saveChanges();
```

Comment:
You are not supposed to save changes during edition when it not required... I see you have the exact same code in the editor bar... is not this one supposed to save/discard tag list changes at the right time??

Reply:
At first, it was decided to save during edition so that you can add/remove tag while still being able to discard changes you've made on the page. After some discussion we come to the conclusion that you may forget yo save during the edition and decided to save the changes if the user forgot to save. Should I just save with the editor button then ?

Reply:
(explained in discord)
- Only save *in database* occurs on editor main save button
- If a tag list was being edited (but not yet "confirmed") when the editor is saved, it is automatically confirmed first thanks to the appropriate code in `cleanForSave`